docs/intake_regionmask.rst

```diff
@@ -0,0 +1,89 @@
+############################
+Using regionmask with intake
+############################
+
+Regions from geopandas shapefiles can be also pre-defined in a yaml file, which can be
+easily shared. This relies on ``intake_geopandas`` and accepts ``regionmask_kwargs``,
+which are passed to ``regionmask.from_geopandas``.
+If you set ``use_fsspec=True`` and use ``simplecache::`` in the url, the shapefile is
+cached locally.
+
+.. ipython:: python
+    :suppress:
+
+    # Use defaults so we don't get gridlines in generated docs
+    import matplotlib as mpl
+    mpl.rcdefaults()
+    mpl.use('Agg')
+
+    # cut border when saving (for maps)
+    mpl.rcParams["savefig.bbox"] = "tight"
+
+You need to install intake_geopandas, which combines geopandas and intake, see
+https://intake.readthedocs.io/en/latest/.
+
+.. ipython:: python
+
+    import intake_geopandas
+    import intake
+    # open a pre-defined remote or local catalog yaml file
+    url = 'https://raw.githubusercontent.com/aaronspring/remote_climate_data/master/master.yaml'
```

Comment:
Would it be difficult to have this catalog in regionmask itself? Then this docu would be independent of changes you do in your repo. You could add it to the docs folder...

Reply:
sure. I could also just create a small catalog in the regionmask, but I want to still access it via http to show that remote catalogs are possible

Reply:
created `data/regions_remote_catalog.yaml`